### DISCUSSION TITLE
Could not poll the database: SQLSTATE[42S02]

### DISCUSSION THREAD

**Discussion by Ultragian** [firefly-iii/firefly-iii#8205] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Brand new re-install of Firefly-III with Cosmos Cloud in Ubuntu. No modifications aside from broadcasting the web ui through my domain using Cloudflare DNS challenge.
The first time I installed it it was working, then I uninstalled to modify code to try and add the Data Importer, after trying multiple times I just decided to wipe and install the raw one available at the Cosmos Cloud store. Now it doesn't even open.

### Debug information

Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

Could not poll the database: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'firefly.configuration' doesn't exist (Connection: mysql, SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

This error occured in file /var/www/html/app/Support/FireflyConfig.php on line 80 with code 0.

### Expected behaviour

Firefly should just open.

### Steps to reproduce

Install Firefly-III from Cosmos Cloud app store using Ubuntu.
Uninstall it.
Wipe data.
Reinstall Firefly-III from Cosmos Cloud app store using Ubuntu.

**Comment by JC5** [comment]
Mmm, I've never worked with the Cosmos Cloud app myself. Do you know who made the app available there?

**Reply by Ultragian** [reply]
![image](https://github.com/firefly-iii/firefly-iii/assets/7107755/b8b4828e-c9c0-496d-bfb0-b3c721e55d9c)
It was the main developer of Cosmos himself. Apparently there is a system in place for requesting new apps for the store, so someone might've asked beforehand. As far as I know it's the main dev himself, according to the text on the right in that image.

**Reply by JC5** [reply]
I suggest you talk to them first.

**Reply by Ultragian** [reply, marked as answer]
Nevermind, somehow I solved it with ChatGPT. Thank you!

**Reply by JC5** [reply]
What was the solution?
